pkg/cms/config/wordpress.go

```diff
@@ -213,9 +214,11 @@ func WriteWordpressConfig(wordpressConfig *model.WordpressConfig, filePath strin
 		return err
 	}
 	// Ensure target directory is writable.
-	os.Chmod(dir, 0755)
+	err = os.Chmod(dir, 0755)
+	util.CheckErr(err)
```

Comment:
it feels like error should just be returned here instead of using `util.CheckErr`, since this function already has an error return value.